sql/core/src/test/java/test/org/apache/spark/sql/JavaColumnExpressionSuite.java

```diff
@@ -0,0 +1,80 @@
+package test.org.apache.spark.sql;
+
+import org.apache.spark.api.java.function.FilterFunction;
+import org.apache.spark.sql.Column;
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.RowFactory;
+import org.apache.spark.sql.test.TestSparkSession;
+import org.apache.spark.sql.types.StructType;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.*;
+
+import static org.apache.spark.sql.types.DataTypes.*;
```

Comment:
Shall we avoid wlidcard import if they are less then 5 imports

Reply:
Hello @HyukjinKwon, I tried to remove the wildcard import and got the following result:

`import java.util.ArrayList;`
`import java.util.Arrays;`
`import java.util.HashSet;`
`import java.util.List;`
`import java.util.Locale;`

`import static org.apache.spark.sql.types.DataTypes.IntegerType;`
`import static org.apache.spark.sql.types.DataTypes.StringType;`
`import static org.apache.spark.sql.types.DataTypes.createArrayType;`
`import static org.apache.spark.sql.types.DataTypes.createStructField;`
`import static org.apache.spark.sql.types.DataTypes.createStructType;`

Is it OK to use wildcard here if exactly 5 imports were used in every case? Thanks.